cluster/gce/trusty/helper.sh

```diff
@@ -30,7 +30,9 @@ source "${KUBE_ROOT}/cluster/gce/debian/helper.sh"
 # $1: template name (required)
 function create-node-instance-template {
   local template_name="$1"
+  sed "s/##KUBELET_PORT##/${KUBELET_PORT}/g" ${KUBE_ROOT}/cluster/gce/trusty/node_template.yaml > ${KUBE_ROOT}/cluster/gce/trusty/node.yaml
```

Comment:
I think the way of using KUBELET_PORT in trusty/node.yaml here is wrong. You don't have to make it a template and 'sed' to set up the correct value (and this will cause GKE code change). The whole logic of current trusty support is as this: (1) It uses the same kube-env generated in debian/helper.sh; (2) node.yaml has a functionality to fetch kube-env from GCE metadata server and parses it. This means that if you need to add a new env variable, you can simply add it in exporting to kube-env like your change to debian/helper.sh. Then, you can directly use ${KUBELET_PORT} in trusty/node.yaml. Please see many examples of using env variable from kube-env in node.yaml.